Makefile.am

```diff
@@ -213,13 +215,27 @@ cpp_server_ct_mirror_LDADD = \
 	$(leveldb_LIBS) \
 	-lprotobuf -lsqlite3
 cpp_server_ct_mirror_SOURCES = \
-  cpp/client/async_log_client.cc \
+        cpp/client/async_log_client.cc \
 	cpp/server/ct-mirror.cc \
 	cpp/server/certificate_handler.cc \
 	cpp/server/handler.cc \
 	cpp/server/json_output.cc \
 	cpp/server/server_helper.cc
 
+cpp_server_ct_mirror_v2_LDADD = \
+	cpp/libcore.a \
+	$(json_c_LIBS) \
+	$(libevent_LIBS) \
+	$(leveldb_LIBS) \
+	-lprotobuf -lsqlite3
+cpp_server_ct_mirror_v2_SOURCES = \
+        cpp/client/async_log_client.cc \
```

Comment:
OK now I fixed it.

On 12 November 2015 at 15:43, Pierre Phaneuf <notifications@github.com>
wrote:

> In Makefile.am
> <https://github.com/google/certificate-transparency/pull/1047#discussion_r44672095>
> :
>
> >  	cpp/server/ct-mirror.cc \
> >  	cpp/server/certificate_handler.cc \
> >  	cpp/server/handler.cc \
> >  	cpp/server/json_output.cc \
> >  	cpp/server/server_helper.cc
> >
> > +cpp_server_ct_mirror_v2_LDADD = \
> > +	cpp/libcore.a \
> > +	$(json_c_LIBS) \
> > +	$(libevent_LIBS) \
> > +	$(leveldb_LIBS) \
> > +	-lprotobuf -lsqlite3
> > +cpp_server_ct_mirror_v2_SOURCES = \
> > +        cpp/client/async_log_client.cc \
>
> I give up, just look at it on GitHub, it's all fairly clear there,
> somehow? cpp_server_xjson_server_SOURCES below looks totally wacky before
> this PR, even (your one change to it seems to be for the better, but the
> other lines are all randomly indented?).
>
> Good luck. [image: :smiley:]
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/google/certificate-transparency/pull/1047/files#r44672095>
> .
>